be/src/Project/Services/Client/Client.Api/Infrastructure/ClientContext.cs

```diff
@@ -16,6 +17,8 @@ protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
 
         public DbSet<Clients.Entities.Client> Clients { get; set; }
         public DbSet<Address> Addresses { get; set; }
-
+        public DbSet<Group> Groups { get; set; }
+        public DbSet<GroupTranslation> GroupTranslations { get; set; }
```

Comment:
**suggestion:** Rename to DbSet<ClientGroupTranslation> ClientGroupTranslations { get; set; }